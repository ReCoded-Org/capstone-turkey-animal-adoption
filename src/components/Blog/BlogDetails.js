// import React from "react";
// import { Container, Row, Col } from "react-bootstrap";

// const BlogDetails = ({ blogContent }) => {
//     console.log(Object.values(blogContent))
//     return (
//         <>
//             {Object.values(blogContent).map((blog) => (
//                 <div>
//                     <p>{blog.publishedDate}</p>
//                     <p>{blog.blogTitle}</p>
//                     <p>{blog.author}</p>
//                     <p>{blog.authorWork}</p>
//                 </div>
//             ))}
//         </>
//     )
// }

// export default BlogDetails