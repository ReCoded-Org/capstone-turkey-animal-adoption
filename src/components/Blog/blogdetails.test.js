import React from "react";
import renderer from "react-test-renderer";
import BlogDetails from "./BlogDetails.js";

it("renders BlogDetails correctly", () => {
  const tree = renderer.create(<BlogDetails />).toJSON();
  expect(tree).toMatchSnapshot();
});
