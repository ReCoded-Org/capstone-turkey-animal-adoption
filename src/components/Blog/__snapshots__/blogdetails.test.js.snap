// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders BlogDetails correctly 1`] = `
<div
  className="mt-5 container"
>
  <div
    className="blog row"
  />
</div>
`;
